src/main/kotlin/com/hibob/academy/resource/SessionResource.kt

```diff
@@ -0,0 +1,43 @@
+package com.hibob.academy.resource
+
+import com.hibob.academy.service.SessionService
+import jakarta.ws.rs.GET
+import jakarta.ws.rs.POST
+import jakarta.ws.rs.Path
+import jakarta.ws.rs.Produces
+import jakarta.ws.rs.core.MediaType
+import jakarta.ws.rs.core.NewCookie
+import jakarta.ws.rs.core.Response
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.RequestBody
+
+@Controller
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/session")
+class SessionResource(private val sessionService: SessionService) {
+
+    companion object {
+        const val COOKIE_NAME = "JWT"
```

Comment:
it would be best to use the same const in both places. that way it will stay consistent

Reply:
I agree, fixed it